hydroshare/settings.py

```diff
@@ -346,7 +347,7 @@
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         'DIRS': [
-            (os.path.join(PROJECT_ROOT, "templates"),)
+            (os.path.join(BASE_DIR, "hs_core", "templates"),)
```

Comment:
PROJECT_ROOT is the "hydroshare" folder with the hydroshare project.  The templates are in "hs_core" not "hydroshare"